### DISCUSSION TITLE
How to obtain a Win32 version of md？

### DISCUSSION THREAD

**Discussion by SaorenPu** [coin-or/Ipopt#767] [Q&A]
For Windows users, What should I do if I now need the Win32 version, as only the Win64 md and mdd are provided in the Releases?
please help me.
![image](https://github.com/coin-or/Ipopt/assets/124563760/15e1378e-770d-467e-9c59-e7369f938e69)

**Comment by svigerske** [comment]
Instructions on building Ipopt are at https://coin-or.github.io/Ipopt/INSTALL.html. I haven't tried whether 32bit systems still work, though.

For some very old Ipopt versions, Windows 32bit builds are still available at https://www.coin-or.org/download/binary/Ipopt/
